tools/src/Patch.ts

```diff
@@ -0,0 +1,24 @@
+import spawnAsync from '@expo/spawn-async';
+
+export async function runPatchAsync(options: {
```

Comment:
As it's just one function, maybe it makes sense to move it to `Utils.ts`? Also, I think "apply" would be better than "run" 😉